src/components/live-tag/live-tag.scss

```diff
@@ -6,6 +6,7 @@
     color: #ffffff;
     font-size: 15px;
     text-transform: uppercase;
+    margin: 0 12px;
```

Comment:
@semarche-kaltura do you want to try use `.playkit-live-tag` class of the original element?
It could be easier for you

Reply:
I tried, too many styles should be overwritten according to design if we use `.playkit-live-tag`
https://app.zeplin.io/project/5d8a2d0fcae74a026ea3adf2/screen/5ea05e833bf3357ed6992db1
https://app.zeplin.io/project/5d8a2d0fcae74a026ea3adf2/screen/5dc005f1cd3d912117742294